spec/lib/interface_spec.rb

```diff
@@ -1,22 +1,60 @@
 require 'spec_helper'
-require 'support/test_interface'
 require 'support/test_implementation'
-require 'support/test_implementation_one'
-require 'support/test_implementation_two'
+require 'support/test_cases'
 require 'interface'
 require 'interface/error/not_implemented_error'
 require 'core_ext/module'
 
 describe Interface do
-  it 'raises an error if the methods are not implemented' do
-    expect { TestImplementation.new }.to raise_error(Interface::Error::NotImplementedError)
-  end
+  describe '#must_implement' do
+    context "when none method implemented" do
+      it "raise method not implemented error from interface" do
+        expect { TestImplementation.new }.to raise_error(Interface::Error::NotImplementedError)
+      end
+    end
+
+    context "when only one right method implemented" do
+      before { TestImplementation.include(MissingOneMethod) }
+
+      it "raise method not implemented error from interface" do
+        expect { TestImplementation.new }.to raise_error(Interface::Error::NotImplementedError)
+      end
+    end
+
+    context "when right methods implemented" do
+      before { TestImplementation.include(RightArity) }
+
+      it "doesn't raise error" do
+        puts TestImplementation.instance_methods.include? :hello_world
```

Comment:
You don't think that it would be better keep it, for safety? It's always helpful if new issues araise!